src/core/sys/posix/sys/ioctl.d

```diff
@@ -323,13 +323,7 @@ else version (OSX)
     import core.sys.posix.termios; // termios
     import core.sys.posix.sys.time; // timeval
 
-    struct winsize
-    {
-        ushort ws_row;
-        ushort ws_col;
-        ushort ws_xpixel;
-        ushort ws_ypixel;
-    }
+    import core.sys.posix.sys.ttycom; // Terminal related ioctls
```

Comment:
On OSX, you would include both ttycom.h and ioctl.h -- I think the separation originated from the fact that for kernel development you would use only ttycom.h, but for userland development you need the ioctl() prototype from ioctl.h.

Like I mentioned in the other comment, I kept the structuring of this to be the same as the .h file structuring so someone reading C documentation on OSX (it's different on Linux though!) knows where to find them.